misc/python/materialize/checks/all_checks/materialized_views.py

```diff
@@ -23,6 +23,9 @@ def initialize(self) -> Testdrive:
                 > CREATE TABLE materialized_views_table (f1 STRING);
                 > INSERT INTO materialized_views_table SELECT 'T1A' || generate_series FROM generate_series(1,10000);
                 > INSERT INTO materialized_views_table SELECT 'T1B' || generate_series FROM generate_series(1,10000);
+
+                # Regression test for #27167.
+                > CREATE MATERIALIZED VIEW zero_arity AS SELECT;
```

Comment:
Will there be any benefit in querying this view in `validate()` or is this not necessary because we only the migration could go wrong?

Reply:
Oh it probably wouldn’t hurt to select from it and make sure it returns nothing! But previously Materialize would fail to boot if you created this view, so just having it in the catalog is sufficient test to protect against the reported issue.

Reply:
Ok, I will add that